packages/web-runtime/src/pages/account.vue

```diff
@@ -18,33 +18,55 @@
       </div>
       <hr />
       <div class="oc-text-bold oc-mb"><translate>Account Information</translate></div>
```

Comment:
Could I interest you in changing this one into a `<h2>` tag for bonus accessibility? :)

Reply:
+1 for that 😁